types/cesium/test/cesium-global.test.ts

```diff
@@ -65,3 +65,24 @@ const label = new Cesium.LabelGraphics({
 
 viewer.scene.globe.tileLoadProgressEvent.addEventListener((progress: number) => progress);
 viewer.scene.globe.terrainProviderChanged.addEventListener((provider: Cesium.TerrainProvider) => provider);
+
+// Cesium3DTileset
+const tileset = viewer.scene.primitives.add(new Cesium.Cesium3DTileset({
+    url: 'http://localhost:8002/tilesets/Seattle/tileset.json',
+    skipLevelOfDetail: true,
+    baseScreenSpaceError: 1024,
+    skipScreenSpaceErrorFactor: 16,
+    skipLevels: 1,
+    immediatelyLoadDesiredLevelOfDetail: false,
+    loadSiblings: false,
+    cullWithChildrenBounds: true
+}));
+
+// Cesium3DTilesetGraphics
+const options = {
+    uri: '0',
+    show: false,
+    maximumScreenSpaceError: 2
+};
+
+ const tilesetModel = new Cesium.Cesium3DTilesetGraphics(options);
```

Comment:
this test is missing from the other test file

Reply:
done, amended. Thanks for review!